pkg/snowflake/tag.go

```diff
@@ -53,6 +54,22 @@ func (tb *TagBuilder) WithSchema(schema string) *TagBuilder {
 	return tb
 }
 
+// WithAllowedValues adds the allowed values to the query
+func (tb *TagBuilder) WithAllowedValues(av []string) *TagBuilder {
+	tb.allowedValues = AvListToString(av)
+	return tb
+}
+
+func AvListToString(avs []string) string {
+	newavs := avs
+	for index, element := range avs {
+		newavs[index] = fmt.Sprintf(`'%v'`, strings.ReplaceAll(element, "'", "\\'"))
```

Comment:
can we create or re-use a helper here instead of relying on what `Sprintf` does?

Reply:
agreed this should be in helpers function